src/opt/vyatta/share/vyatta-op/functions/interpreter/vyatta-unpriv

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
This is also certainly a highly non-trivial file that requires license info and attribution.

Reply:
Of course - attribution added